### PR TITLE
Fsa22 v2 284 refactor rename breakpoint sass variables

### PR DESCRIPTION
**Pull request by tisidro**
### Description:

Sass breakpoint variables are updated throughout the code base to use descriptive names rather than device-type names.

### Spec:

Designs: [Designs](DESIGN_URL)

See Story: [FSA22V2-284](https://sparkbox.atlassian.net/browse/FSA22V2-284)

### Validation:

- [ ] This PR has code changes, and our linters still pass.
- [ ] This PR affects production code, so it was browser tested (see below).
- [ ] This PR has new code, so new tests were added or updated, and they pass.

#### To Validate:

1. Make sure all PR Checks have passed (GitHub Actions, CircleCI, Code Climate, etc).
2. Pull down all related branches.
3. Run `npm install` to make sure you have all proper dependencies. (this project requires Node 16+)
4. Copy the `.env.example` file, making sure it's in the root of your project, and rename it `.env.local`. Search for "Accessible Components" in 1Password to find the API Key and Base ID. Add both of those secrets to your newly created `.env.local` file. That should establish your connection to our Airtable database.
5. Run `npm run lint` to confirm no errors.
6. Run `npm run test` to confirm all tests pass.
7. In terminal: `npm run dev`.
8. Check [localhost:3000](http://localhost:3000/) to see changes.
9. Run axe Devtools on affected pages to confirm no urgent errors.

In the styles scss files, check that any breakpoint names make sense in context of what they are accomplishing for the display.

### Pending Questions:

1.
2.

---

#### Browser Testing

In these browsers, behavior & design closely match original specifications. A user is able to access all content and functionality, including the usability of required assistive devices, such as keyboard and screenreader.

**macOS**

- [ ] Safari (last 2 major versions)
- [ ] Chrome (last 6 months)
- [ ] Firefox (last 6 months)

**Windows**

- [ ] Chrome (last 6 months)
- [ ] Firefox (last 6 months)
- [ ] Edge (last 6 months)

**Mobile**

- [ ] Safari (last 2 major versions)
- [ ] Chrome on Android (last 6 months)
- [ ] Firefox on Android (last 6 months)

### CODE REVIEW

File: src/styles/components/_detailsBanner.scss [hunk 1]
```diff
@@ -15,7 +13,7 @@ $breakpoint-tablet: 820px;
     font-family: Viga, sans-serif;
     font-size: 1.5625rem; // 25px/16
 
-    @media only screen and (min-width: $breakpoint-tablet) {
+    @media (min-width: $breakpoint-increase-sizes) {
```
Comment (jkohrt7) [inline src/styles/components/_detailsBanner.scss:16]: To me, the meaning of $breakpoint-increase-sizes isn't super clear. For this file, could you do something similar to code-block.scss and rename it to `$breakpoint-increase-font-size` or something similar?

**Review by jkohrt7 — DISMISSED** [review]
Looks good! The breakpoints make a lot more sense at a glance now. The was one variable name I thought could be clearer, but it might just be personal preference--feel free to leave it as-is if you disagree 😄

File: src/styles/components/_detailsBanner.scss (hunk 1 above)
Comment (tisidro) [reply src/styles/components/_detailsBanner.scss:16]: This actually came out of pairing w/ Zastrow and I had the same feeling--but he pointed out that if you just say "increase sizes" you can address that it increases font size/padding/margin. Some of the media queries do more than one thing. But I did tinker w/ the idea of specifically saying "increase-font-size" if that is all it does. I'm going to go through and see if any are specific to that and rename those more specifically. Thanks Jared! :)

Comment (dustin-jw) [reply src/styles/components/_detailsBanner.scss:16]: Maybe `$bp-scale-up` or something like that? I feel like that gives an idea that everything sort of gets bigger at that size.

Comment (dustin-jw) [reply src/styles/components/_detailsBanner.scss:16]: As is usually the case in development, it depends 😄 
If the breakpoint affects a variety of things, then a blanket name that describes the overall effect makes sense, but if it literally just changes the font size, then a more specific name works too.

Comment (tisidro) [reply src/styles/components/_detailsBanner.scss]: Thanks, I did this update & pushed the changes. Osmond will review :)

**Review by arnest00 — DISMISSED** [review]
Looks good, @tisidro!!

File: src/styles/components/_code-block.scss [hunk 2]
```diff
@@ -50,7 +52,7 @@ $breakpoint-tablet: 820px;
     margin-top: 1rem;
     border: 2px solid var(--foreground-color);
 
-    @media only screen and (min-width: $breakpoint-tablet) {
+    @media (min-width: $breakpoint-scale-up) {
```
Comment (jkohrt7) [inline src/styles/components/_code-block.scss:55]: Nice! The new 'scale-up' name was a good choice.

**Review by jkohrt7 — DISMISSED** [review]
Looks good!

**Review by jkohrt7 — DISMISSED** [review]
Make sure you squash before merging in now that you've added another commit!

**Pull request merged**